identity-apps-core/apps/authentication-portal/src/main/webapp/WEB-INF/web.xml

```diff
@@ -462,6 +467,11 @@
         <url-pattern>/device_success.do</url-pattern>
     </servlet-mapping>
 
+    <servlet-mapping>
+        <servlet-name>ciba_success.do</servlet-name>
+        <url-pattern>/ciba_success.do</url-pattern>
```

Comment:
This will make `ciba_success` url appear in the end user's browser. Showing an unnecessary tech term wouldn't look good IMO. Shall we use a generic yet relevant name instead?